### PR TITLE
feat: replace message extension with message extension action template

### PR DESCRIPTION
**Pull request by tecton**
Related task: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/24719427

### CODE REVIEW

File: packages/fx-core/src/component/coordinator/index.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ export enum TemplateNames {
   CommandAndResponse = "command-and-response",
   Workflow = "workflow",
   DefaultBot = "default-bot",
-  MessageExtension = "message-extension",
+  MessageExtension = "message-extension-action",
```
Comment (yukun-dong) [inline packages/fx-core/src/component/coordinator/index.ts:84]: How about renaming this variable to "MessageExtensionAction" to avoid confusion.

Comment (tecton) [reply packages/fx-core/src/component/coordinator/index.ts:84]: Sure, will fix.

File: packages/fx-core/src/component/coordinator/index.ts [hunk 2]
```diff
@@ -107,7 +107,7 @@ const Feature2TemplateName: any = {
   [`${CapabilityOptions.commandBot().id}:undefined`]: TemplateNames.CommandAndResponse,
   [`${CapabilityOptions.workflowBot().id}:undefined`]: TemplateNames.Workflow,
   [`${CapabilityOptions.basicBot().id}:undefined`]: TemplateNames.DefaultBot,
-  [`${CapabilityOptions.me().id}:undefined`]: TemplateNames.MessageExtension,
+  [`${CapabilityOptions.me().id}:undefined`]: TemplateNames.MessageExtensionAction,
```
Comment (yukun-dong) [inline packages/fx-core/src/component/coordinator/index.ts:110]: Based on below, should here be `CapabilityOptions.collectFormMe().id` ?https://github.com/OfficeDev/TeamsFx/blob/dea6b9e8aae05e82dca0dedecfadd0fd22ea591f/packages/fx-core/src/question/create.ts#L391

I think the reason it still works is that `CapabilityOptions.collectFormMe()` and `CapabilityOptions.me()` share the same id.

Comment (tecton) [reply packages/fx-core/src/component/coordinator/index.ts:110]: Yes, I agree. Thanks.

**Review by yukun-dong — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**